text/0000-javascript-module-api.md

```diff
@@ -0,0 +1,1151 @@
+| `Ember.Checkbox`                      | `import Checkbox from "@ember/component/checkbox"`                         |
+| `Ember.Component`                     | `import Component from "@ember/component"`                                 |
+| `Ember.ContainerDebugAdapter`         | `import ContainerDebugAdapter from "@ember/debug/container-debug-adapter"` |
+| `Ember.Controller`                    | `import Controller from "@ember/controller"`                               |
+| `Ember.DataAdapter`                   | `import DataAdapter from "@ember/debug/data-adapter"`                      |
+| `Ember.DefaultResolver`               | `import GlobalsResolver from "@ember/application/globals-resolver"`        |
+| `Ember.Enumerable`                    | `import Enumerable from "@ember/enumerable"`                               |
+| `Ember.Evented`                       | `import Evented from "@ember/object/evented"`                              |
+| `Ember.HashLocation`                  | `import HashLocation from "@ember/routing/hash-location"`                  |
+| `Ember.Helper`                        | `import Helper from "@ember/component/helper"`                             |
+| `Ember.Helper.helper`                 | `import { helper } from "@ember/component/helper"`                         |
+| `Ember.HistoryLocation`               | `import HistoryLocation from "@ember/routing/history-location"`            |
+| `Ember.Location`                      | `import Location from "@ember/routing/location"`                           |
+| `Ember.Map`                           | `import EmberMap from "@ember/map"`                                        |
+| `Ember.MapWithDefault`                | `import MapWithDefault from "@ember/map/with-default"`                     |
+| `Ember.Mixin`                         | `import Mixin from "@ember/object/mixin"`                                  |
+| `Ember.MutableArray`                  | `import MutableArray from "@ember/array/mutable"`                          |
+| `Ember.NoneLocation`                  | `import NoneLocation from "@ember/routing/none-location"`                  |
+| `Ember.Object`                        | `import EmberObject from "@ember/object"`                                  |
+| `Ember.RSVP`                          | `import RSVP from "rsvp"`                                                  |
+| `Ember.Resolver`                      | `import Resolver from "@ember/application/resolver"`                       |
+| `Ember.Route`                         | `import Route from "@ember/routing/route"`                                 |
+| `Ember.Router`                        | `import Router from "@ember/routing/router"`                               |
+| `Ember.Service`                       | `import Service from "@ember/service"`                                     |
+| `Ember.String.camelize`               | `import { camelize } from "@ember/string"`                                 |
```

Comment:
@kellyselden Yes, you are mistaken. :) http://rollupjs.org/?version=0.41.2&shareable=JTdCJTIyb3B0aW9ucyUyMiUzQSU3QiUyMmZvcm1hdCUyMiUzQSUyMmVzJTIyJTJDJTIybW9kdWxlTmFtZSUyMiUzQSUyMm15QnVuZGxlJTIyJTJDJTIyZ2xvYmFscyUyMiUzQSU3QiU3RCUyQyUyMm1vZHVsZUlkJTIyJTNBJTIyJTIyJTdEJTJDJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmltcG9ydCUyMCU3QiUyMGNhbWVsaXplJTIwJTdEJTIwZnJvbSUyMCU1QyUyMi4lMkZzdHJpbmcuanMlNUMlMjIlM0IlNUNuJTVDbmNvbnNvbGUubG9nKGNhbWVsaXplKCU1QyUyMmhlbGxvJTIwd29ybGQlNUMlMjIpKSUzQiUyMiU3RCUyQyU3QiUyMm5hbWUlMjIlM0ElMjJzdHJpbmcuanMlMjIlMkMlMjJjb2RlJTIyJTNBJTIyZXhwb3J0JTIwJTdCJTIwZGVmYXVsdCUyMGFzJTIwY2FtZWxpemUlMjAlN0QlMjBmcm9tJTIwJTVDJTIyLiUyRmNhbWVsaXplLmpzJTVDJTIyJTNCJTVDbmV4cG9ydCUyMCU3QiUyMGRlZmF1bHQlMjBhcyUyMGRhc2hlcml6ZSUyMCU3RCUyMGZyb20lMjAlNUMlMjIuJTJGZGFzaGVyaXplLmpzJTVDJTIyJTNCJTIyJTdEJTJDJTdCJTIybmFtZSUyMiUzQSUyMmNhbWVsaXplLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmV4cG9ydCUyMGRlZmF1bHQlMjBmdW5jdGlvbiUyMGNhbWVsaXplKCklMjAlN0IlNUNuJTdEJTIyJTdEJTJDJTdCJTIybmFtZSUyMiUzQSUyMmRhc2hlcml6ZS5qcyUyMiUyQyUyMmNvZGUlMjIlM0ElMjJleHBvcnQlMjBkZWZhdWx0JTIwZnVuY3Rpb24lMjBkYXNoZXJpemUoKSUyMCU3QiU1Q24lN0QlMjIlN0QlNUQlN0Q=